scripts/dockerNodeStats/main.py

```diff
@@ -0,0 +1,197 @@
+import csv
+import json
+import logging
+import os
+import subprocess
+from collections import defaultdict
+
+import tqdm
+
```

Comment:
I would propose that we start trying that library:
https://github.com/amoffat/sh

6.7Kstars... seems pretty cool for wrapping these subprocesses... found it through some articles.

Reply:
I would avoid executing bash in Python when it is possible

Reply:
I intriduced `sh`. @YuryHrytsuk you are absolutely right, this was based on some script by an AI-Bot, who already used a lot of shell-calls, and was provided to me by @mguidon . If we code something properly ourselves we should indeed avoid console calls and use packages, modules and APIs :)